techniques/systemSettings/systemManagement/fstabConfiguration/4.0/metadata.xml

```diff
@@ -42,13 +42,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     <SECTION name="Mountpoint" multivalued="true">
       <SELECT1>
         <NAME>FSTAB_ALTERONLY</NAME>
+        <DESCRIPTION>Mountpoint action</DESCRIPTION>
```

Comment:
Shouldn't we be talking about state instead of action? In the spirit of desired state